src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -245,17 +245,17 @@
     // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
     const char *appctx_keys [] = {
         "RUNTIME_IDENTIFIER", 
+        "APP_CONTEXT_BASE_DIRECTORY",
 #ifndef INVARIANT_GLOBALIZATION
-        "ICU_DAT_FILE_PATH",
+        "ICU_DAT_FILE_PATH"
 #endif
-        "APP_CONTEXT_BASE_DIRECTORY"
     };
     const char *appctx_values [] = {
         APPLE_RUNTIME_IDENTIFIER,
+        bundle,
 #ifndef INVARIANT_GLOBALIZATION
-        icu_dat_path,
+        icu_dat_path
```

Comment:
nit: trailing commas are not an error.

(and they play nicer with git history when someone adds the next array entry)